setup.py

```diff
@@ -34,30 +34,45 @@
     from distutils.core import setup, Extension
     from distutils.command.build_ext import build_ext
 
+import subprocess

```

Comment:
Does this file need to be changed? It looks like some of it is particular to what you were doing (`repourl = 'https://github.com/Vadiml1024/python-libarchive'`)

Reply:
Done